screens/buyer_search.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+
+export default class App extends React.Component {
```

Comment:
Change the class name from `App` to `BuyerSearch`.

Remember that you are going to use this class as a component somewhere else in the code. It can't have the same name as the class in `App.js`